Makefile

```diff
@@ -6,6 +6,10 @@
 # Ensure go-extpoints and go-import-subtree are available for go generate
 export PATH := $(GOPATH)/bin:$(PATH)
 
+bootstrap:
```

Comment:
Sorry, I was a lot less than clear with this comment.  What I'm referring to is the "bootstrapping" that is done in the travis environment and I wonder i the same needs to be done here in this make target.